### PR TITLE
Add TypeScript full course for beginners

### PR DESCRIPTION
**Pull request by muhammadcaeed**
Added a full course of TypeScript for beginners that covers the latest updates till 2021.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -881,7 +882,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Typescript - The Basics](https://www.youtube.com/watch?v=ahCwqrYpIuM)
 - [Typescript Basics - Course](https://www.youtube.com/playlist?list=PLqq-6Pq4lTTanfgsbnFzfWUhhAz3tIezU)
 - [Clean Code concepts adapted for TypeScript](https://github.com/labs42io/clean-code-typescript)
-
+- [TypeScript Course for Beginners 2021 - Learn TypeScript from Scratch!](https://www.youtube.com/watch?v=BwuLxPH8IDs&t=1s)
```
Comment (shahednasser) [inline README.md:885]: Please add a new empty line after this

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @muhammadcaeed for your contribution. I have added one comment for you to fix. I have also added the `hacktoberfest-accepted` label in case this PR doesn't get merged before the end of today.

**Comment by muhammadcaeed** [comment]
@shahednasser  You might notice an extra commit for another section, but it's already reverted. I will add a new PR for it.

**Comment by shahednasser** [comment]
Thank you!

**Pull request merged**